samples/idea-latest.yaml

```diff
@@ -0,0 +1,27 @@
+kind: DevWorkspace
+apiVersion: workspace.devfile.io/v1alpha2
+metadata:
+  name: idea-latest
+spec:
+  started: true
+  template:
+    projects:
+      - name: web-nodejs-sample
+        git:
+          remotes:
+            origin: "https://github.com/che-samples/python-hello-world.git"
+          checkoutFrom:
+            revision: "devfilev2"
```

Comment:
don't do checkout to **devfilev2**, this branch is going to be removed https://github.com/eclipse/che/issues/21983
just use the default **main** branch

Reply:
Done